constructor/construct.py

```diff
@@ -792,6 +801,15 @@ def verify(info):
                 types_str = " or ".join([type_.__name__ for type_ in types])
                 sys.exit(f"Value for 'extra_envs.{env_name}.{key}' "
                          f"must be an instance of {types_str}")
+    if signtool := info.get("windows_signing_tool"):
+        if signtool not in WIN_SIGNTOOLS:
+            sys.exit(
```

Comment:
This reminds me we need to start raising exceptions instead of sys.exits, but that's for another PR.